lib/sfn/command_module/template.rb

```diff
@@ -112,7 +112,7 @@ def load_template_file(*args)
                 if(formation.compile_state)
                   current_state = current_state.merge(formation.compile_state)
                 end
-                ui.info "#{ui.color('Compile time parameters:', :bold)} - template: #{ui.color(formation.name, :green, :bold)}"
+                ui.info "#{ui.color('Compile time parameters:', :bold)} - template: #{ui.color(formation.name, :green, :bold)}" unless(config[:print_only])
```

Comment:
Can you remove the parens from the inline condition: `unless config[:print_only]`